pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix

```diff
@@ -0,0 +1,26 @@
+# Webcord with Vencord.
+# Utilizes the Vencord Chrome extension.
```

Comment:
If this is of relevance, please move it to the description

Reply:
added to description